images/kindnetd/cmd/kindnetd/main.go

```diff
@@ -70,12 +70,8 @@ func main() {
 	// enforce ip masquerade rules
 	// TODO: dual stack...?
 	masqAgent := NewIPMasqAgent(net.IsIPv6String(hostIP), []string{os.Getenv("POD_SUBNET")})
-	go func() {
-		// TODO: use logging and continue retrying instead...
-		if err := masqAgent.SyncRulesForever(time.Second * 60); err != nil {
-			panic(err)
-		}
-	}()
+	// TODO: handle from the main control loop once this moves to a proper controller
+	go masqAgent.SyncRulesForever(time.Second * 60)
```

Comment:
this is _technically_ fine but is better to still be wrapped in a `func() { }()`.

the reason for this is that arguments are evaluated immediately, which can be a major footgun in the future, it's probably best to go ahead and enclose this in it's own scope now.